ui/src/components/form/endpoint_combo_box/SelectEndpointComboBox.js

```diff
@@ -4,12 +4,13 @@ import { EuiComboBoxSelect } from "../combo_box/EuiComboBoxSelect";
 
 export const SelectEndpointComboBox = ({
   value,
+  protocol,
   onChange,
   options,
   ...props
 }) => {
   const onCreateOption = (value) => {
-    if (!isValidUrl(value)) {
+    if (!isValidUrl(value, protocol)) {
```

Comment:
actually i dont grasp the rationale for this behavior. If url is not valid, the router object values is never set. So if it is non empty, means it validated.

I think we can remove this and change and let the validation be done in `schema.js` to prevent two validation logic. 

Do lmk if i missed anything to why this is needed here
```
onCreateOption={onCreateOption}
onCreateOption={e => onChange(e)}
```

Reply:
>  checking free-form text upfront in cases like these can be useful.
fair enough, will keep this then. I didn't see other free-form input component having such validation so I thought this was off, but your right to say it could a tiny enhancement to others rather than removing this. Thanks!